pkgs/by-name/ma/maid/test.nix

```
@@ -0,0 +1,18 @@
+{ runCommandLocal, maid }:
+
+runCommandLocal { nativeBuildInputs = [ maid ]; }
+  ''
```

```suggestion
runCommandLocal "test-maid-run" {
  nativeBuildInputs = [ maid ];
} ''
```

The positional argument for package name is required when calling `runCommandLocal`.